src/components/cvaa-overlay/custom-captions-window.js

```diff
@@ -35,6 +35,16 @@ class CustomCaptionsWindow extends Component {
     this.props.changeCaptionsStyle(this.props.customTextStyle);
   };
 
+  /**
+   * transition to state handler
+   *
+   * @returns {void}
+   * @memberof MainWindow
+   */
+  transitionToState = (): void => {
```

Comment:
why you re-declare the function here if you are not using it inside this component ?

Reply:
I'm using it inside the onClick function, line 172

Reply:
you can just call this.props.transitionToState(this.props.cvaaOverlayState.Main); there

Reply:
we did the same with this.changeCaptionsStyle(), wrap it and then use it so i went on the same way